internal/bminventory/inventory_test.go

```diff
@@ -8321,36 +8321,6 @@ var _ = Describe("Upload and Download logs test", func() {
 		reply := bm.V2UploadLogs(ctx, params)
 		Expect(reply).Should(BeAssignableToTypeOf(installer.NewV2UploadLogsNoContent()))
 	})
-	It("start collecting hosts logs indication", func() {
```

Comment:
These tests must be preserved in their v2 version. Are there parallel tests for v2 already? If not, they should be converted

Reply:
Yes.
https://github.com/openshift/assisted-service/blob/07f48bf9e875bee43df80a928210fe6a615ddb7a/internal/bminventory/inventory_test.go#L8355-L8384